packages/jaeger-ui/src/model/transform-trace-data.tsx

```diff
@@ -32,6 +33,24 @@ function deduplicateTags(spanTags: Array<KeyValuePair>) {
   return { tags, warnings };
 }
 
+export function orderTags(spanTags: Array<KeyValuePair>, topPrefixes: Array<string>) {
+  const orderedTags: Array<KeyValuePair> = [...spanTags];
```

Comment:
It seems as though `spanTags.slice()` is faster than `[...spanTags]`
[source](https://measurethat.net/Benchmarks/ShowResult/84928)